### DISCUSSION TITLE
スコアについてのご意見・ご感想をお聞かせください！

### DISCUSSION THREAD

**Discussion by UserSuccessTeamLAPRAS** [lapras-inc/lapras-user-community#12] [スコアについてのご意見・議論]
技術力スコアについて、皆様のご意見をお聞かせください！

## 2023/2/6より、EXP機能が登場しました！ 🎉 
Qiita/ Zennの記事を作成することでEXPを獲得することができ、より自分自身の積み上げが目に見える形でわかるようになりました。

ぜひ皆様の感想・ご意見をお寄せください！

**Comment by motuo1201** [comment]
やっぱり、自分のスコアが市場においてどれくらいのところにいるか、というのは知りたいと思います！
あと、スコアが上がった時の自分自身へのインセンティブなんかが分かりやすくなるとうれしいなぁ、と思ったりします。

もともとのスコアは
> スコアは平均値が3.0、目安は上位13% 3.3以上、4% 3.5以上です。

とかあったのでわかりやすかったですねー

**Reply by UserSuccessTeamLAPRAS** [reply]
ご意見ありがとうございます！自分自身へのインセンティブ、大事ですよね

**Comment by mskasa** [comment]
本機能の導入で、Zennへの投稿のモチベーションが上がりました！
ありがとうございます。

技術力スコアの算出方法につきまして、僭越ながら意見を述べさせていただきます。

１．GitHub Repoの重み付けが圧倒的に大きい
これですと、技術力スコアにおいて、
アプリケーションエンジニア > インフラエンジニア
の傾向が強く出てしまうのではないでしょうか。
もしその傾向があるのでしたら、重み付けについて再考願いたいです。

２．teratail Score
アカウントすら持っていない人はこの項目によって技術力スコアが下がってしまっているみたいなことはありますでしょうか。
あるのでしたらこの項目について再考願いたいです（現在、teratailとの連携が出来ない状態のようですし…）。

以上、ご確認お願いします。
LAPRASはとても好きなサービスなので、さらなる発展を期待しております！

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRAS運営です。
嬉しいご意見ありがとうございます！早速社内でシェアさせていただきました。
ユーザーの皆様のモチベーション向上のお役に立てているというのはとても嬉しいです！

今後もアップデートを予定していますので、是非またご意見・ご感想をお待ちしています 😄

**Comment by rascal-3** [comment]
運営ご担当者様

GitHubのプライベートリポジトリの情報の設定をしても、反映されません。
具体的に例を申し上げますと、5月以降のアクティビティ(いわゆる草と呼ばれるもの)が読み込まれていないようです。
過去分についても同様です。

何か他に設定すべき点がありましたらご教示頂けますと幸いです。
宜しくお願い申し上げます。

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASユーザーサポートです。

GitHub のプライベートリポジトリに関するアクションにつきましては、Activity Log には表示がされません。
技術力スコアにのみ反映される仕様となっております。

改善のご要望として開発チームに共有させていただきます。

今後ともLAPRASをよろしくお願いします。

**Reply by rascal-3** [reply]
ご返信ありがとうございます。
スコアについても反映されていないようなのですが…

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASユーザーサポートです。

Private contributions につきまして、9月13日以前のものはすでに技術力スコアに反映されておりました。
9月13日以降の分についても、お問い合わせいただいた後に反映されましたので、現在技術力スコアは最新の状態になっております。

スコアへ活動が反映されていないように感じられましたら、Portfolio画面（https://lapras.com/person）右上の「更新」ボタンを押して頂きますと、クロールと計算が行われ、長くとも24時間以内に更新が完了します。
また、 Private contributions が追加されても、大きくスコアに影響しない場合もございます。

この度はご不便おかけしました。今後ともLAPRASをよろしくお願いします。

**Reply by rascal-3** [reply]
ご確認ありがとうございます。

度々恐れ入りますが、リポジトリの数が実際と異なっているのですが、プライベートリポジトリは表示されない仕様なのでしょうか。
Portfolio画面の右側のGitHubの部分にも該当するリポジトリはありませんでした。

宜しくお願い致します。

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASユーザーサポートです。

GitHub のプライベートリポジトリに関するアクションにつきましては、現状技術力スコアにのみ反映される仕様となっております。
そのため、Portfolio画面上や EXP 等にも反映がされません。

引き続きLAPRASをよろしくお願いします。

**Reply by rascal-3** [reply]
度々申し訳ありません。

主にPrivateリポジトリで活動を行っており、その間はやはりポイントが変わらないのですが、
Publicなリポジトリを追加し少し活動を行ったところ、0.01ポイント増加しておりました。

設定はガイダンスの通りにしているのですが、他に何か注意点はあるでしょうか。

何卒宜しくお願い申し上げます。

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASユーザーサポートです。

確認したところ、GitHubの設定等は問題ありませんでした。
Private contributions を追加されても、大きくスコアに影響しない場合もございます。

今後ともLAPRASをよろしくお願いします。

**Comment by tockri** [comment]
Zennの連携を設定したのですが、Zennに書いているBookの記事がPortfolio画面に表示されません。
もしかして、Bookを公開していても記事としてカウントされていないのでしょうか？

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASユーザーサポートです。

bookにつきましては、Zennのセクションには表示がされない仕様となりますが、Activity Log には反映される仕様になっております。

今後ともLAPRASをよろしくお願いします。

**Comment by nunukim** [comment]
今後、ご意見・ご感想はカテゴリ「[📈スコアについてのご意見・議論](https://github.com/lapras-inc/lapras-user-community/discussions/categories/%E3%82%B9%E3%82%B3%E3%82%A2%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6%E3%81%AE%E3%81%94%E6%84%8F%E8%A6%8B-%E8%AD%B0%E8%AB%96)」への新規Discussion として、または関連するDiscussion のコメントとして投稿いただく形とすることにしました。

こちらの Discussion はCloseさせていただきます 🙏🏻
